scala_zio_bench/src/main/scala/com/example/helloworld/GreeterServer.scala

```diff
@@ -1,10 +1,45 @@
 package com.example.helloworld
 
-import scalapb.zio_grpc.{ServerMain, ServiceList}
+import scalapb.zio_grpc.{Server, ServerLayer, ServerMain, ServiceList}
+import zio.ZLayer
+
+import java.util.concurrent.Executors
 
 object GreeterServer extends ServerMain {
   def services: ServiceList[Any] = ServiceList.add(GreeterImpl)
 
   // Default port is 9000
   override def port = 50051
+
+  override def serverLive: ZLayer[Any, Throwable, Server] = {
+    val sb = builder
+
+    /**
+     * Allow customization of the Executor with two environment variables:
+     *
+     * <p>
+     * <ul>
+     * <li>JVM_EXECUTOR_TYPE: direct, workStealing, single, fixed, cached</li>
+     * <li>JVM_EXECUTOR_THREADS: integer value.</li>
+     * </ul>
+     * </p>
+     *
+     * The number of Executor Threads will default to the number of
+     * availableProcessors(). Only the workStealing and fixed executors will use
+     * this value.
+     */
+    val threads = System.getenv("JVM_EXECUTOR_THREADS")
```

Comment:
In general, should this be rather `GRPC_SERVER_CPUS`? This is the variable we pass to servers so that they know how many will be available upfront. Or, will the `JVM_EXECUTOR_THREADS` correctly do that under the hood? Unless there is some JVM magic happening, the service will assume max CPUs on the host (e.g., 32), use a proper GC for that but then it'll get clamped at, e.g., 1. This may also be the case for other JVM-based implementations here.

Reply:
Yeah I don't think there's any magic, so it will use `Runtime.getRuntime.availableProcessors`. I just copied the way it was done in the fs2 one, it seems to be used that way in 4 different tests: https://github.com/search?q=repo%3ALesnyRumcajs%2Fgrpc_bench%20JVM_EXECUTOR_THREADS&type=code
Shall I change them all to use `GRPC_SERVER_CPUS` instead?

Reply:
I just changed it.